README.md

```diff
@@ -183,7 +183,7 @@ Thanks to all <img src="https://assets-global.website-files.com/5dbb30f00775d4c3
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍

Reply:
All right, thanks for the advice! 👍